commons-okhttp/src/test/scala/com/avsystem/commons/okhttp/rest/UsesOkHttpClient.scala

```diff
@@ -0,0 +1,9 @@
+package com.avsystem.commons
+package okhttp.rest
+
+import okhttp3.OkHttpClient
+import org.scalatest.{BeforeAndAfterAll, Suite}
+
+trait UsesOkHttpClient extends BeforeAndAfterAll { this: Suite =>
```

Comment:
If there's no `beforeAll` / `afterAll` setup/destroy logic then I don't see a reason for introducing entire separate trait.